setup.py

```diff
@@ -24,22 +24,37 @@
 import imp
 import io
 import os
+import sys
+
 from setuptools import setup, find_packages
 
 version = imp.load_source(
     'databricks_cli.version', os.path.join('databricks_cli', 'version.py')).version
 
+install_requires = [
+    'click>=6.7',
+    'requests>=2.17.3',
+    'tabulate>=0.7.7',
+    'six>=1.10.0'
+]
+
+if sys.version_info < (3, 6):
+    install_requires += ['configparser>=0.3.5;python_version < "3.6"']
+else:
+    install_requires += [
+        # additional deps for tunneling
+        'aiohttp>=3.7.4',
+        'cryptography>=3.4.6',
+        'python-socketio>=5.0.4',
+        'tenacity>=7.0.0',
+        'tornado>=6.0.1',
```

Comment:
It's generally a good practice to pin these to the major version, for e.g., <7 for tornado to avoid getting breaking changes

Reply:
hm.. pinning is a little tricky cos if we're being too restrictive then it might break user workflow (eg: user having different pkg dep versions in their env). so it'd be good to use a good deps version range that don't break the deps for the cli while also not being too restrictive.

Reply:
Yea, I'm not suggested we pin to the patch version, just to the next major version. For e.g., `tornado>=6.01 < 7`

Reply:
ok, i think we have two options here:
1. pin the upper limit to the next major version
2. pin the upper limit based on request / when we discover some version is not compatible

for 2, i wonder f it's possible that the user is using the next major version that is still compatible but not sure how common it is.

Reply:
I prefer 2), because more often that not things can break unexpectedly across major versions.